File: Introduccion a Poo/Ejercicio 9/Java/Computadora.java
public class Computadora {
    private String estado;
    public Computadora() {
        estado = "apagada";
    }
    public void mostrarComponentes() {
        System.out.println("Procesador, RAM, Disco Duro, Tarjeta Madre");
    }
    public void encender() {
        estado = "encendida";
        System.out.println("La computadora está encendida");
    }
    public void apagar() {
        estado = "apagada";
        System.out.println("La computadora está apagada");
    }
}

File: Introduccion a Poo/Ejercicio 9/Python/Computadora.py
class Computadora:
    def __init__(self):
        self.estado = "apagada"
    def componentes(self):
        print("Procesador, RAM, Disco Duro, Tarjeta Madre")
    def encender(self):
        self.estado = "encendida"
        print("La computadora está encendida")
    def apagar(self):
        self.estado = "apagada"
        print("La computadora está apagada")
c = Computadora()
c.componentes()
c.encender()
c.apagar()
